src/handlers/api/types.ts

```diff
@@ -16,10 +16,19 @@ export const nonEmptyWhitespaceFreeString: D.Decoder<
   )
 );
 
-export const lock = D.struct({
-  name: nonEmptyWhitespaceFreeString,
-  owner: nonEmptyWhitespaceFreeString,
-});
+export const lock = pipe(
+  D.struct({
+    name: nonEmptyWhitespaceFreeString,
+    owner: nonEmptyWhitespaceFreeString,
+  }),
+  // Making message field optional for backward compatibility
```

Comment:
You can remove this comment
_It's not just for backwards compatibility, people also might not chose to add a description - so it's always optional_